mod/au_subgroups/views/default/au_subgroups/search_results.php

```diff
@@ -24,7 +24,7 @@
 }
 else {
   $query = sanitize_string($vars['q']);
-  $options['wheres'][] = "g.name LIKE '{$query}%'";
+  $options['wheres'][] = "g.name LIKE '%{$query}%'";
```

Comment:
If we want to do something like that, we need to think when someone look for something in french...